### PR TITLE
TOMEE-2872 Translate to Portuguese: examples/mp-rest-jwt

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira: https://issues.apache.org/jira/browse/TOMEE-2872

### CODE REVIEW

**Comment by cesarhernandezgt** [comment]
+1 in format, thank you @Daniel-Dos
Just missing now a Portuguese review.

File: examples/mp-rest-jwt/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    @Produces
+    JWTAuthContextInfo getContextInfo() throws InvalidKeySpecException, NoSuchAlgorithmException {
+        return getOptionalContextInfo().get();
+    }
+}
+----
+
+== Usando o MicroProfile JWT no TomEE
+
+O recurso JAX-RS `MoviesRest.java` contém vários pontos de extremidade seguros, que são alcançados usando a anotação padrão `@ RolesAllowed`. O MicroProfile JWT é responsável por validar solicitações recebidas com o cabeçalho `Authorization' que fornece um token de acesso assinado.
```
Comment (willesreis) [inline examples/mp-rest-jwt/README_pt.adoc:72]:
```suggestion
O recurso JAX-RS `MoviesRest.java` contém vários endpoints que estão protegidos usando a anotação padrão `@ RolesAllowed`. O MicroProfile JWT é responsável por validar solicitações recebidas com o cabeçalho `Authorization' que fornece um token de acesso assinado.
```

File: examples/mp-rest-jwt/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+ @Inject
+ @ConfigProperty(name = "java.runtime.version")
+ private String javaVersion;
+ 
+----
+
+== Sobre a arquitetura de teste
+
+Os casos de teste para este projeto são construídos com o Arquillian. A configuração Arquillian pode ser encontrada em `src/test/resources/arquillian.xml`
```
Comment (willesreis) [inline examples/mp-rest-jwt/README_pt.adoc:146]:
```suggestion
Os casos de teste para este projeto são construídos com o Arquillian. A configuração do Arquillian pode ser encontrada em `src/test/resources/arquillian.xml`
```

File: examples/mp-rest-jwt/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+A classe `TokenUtils.java` é usada durante o teste para atuar como um servidor de Autorização que gera `Access Tokens` com base nos arquivos de configuração `privateKey.pem`,` publicKey.pem`, `Token1.json` e` Token2 .json`.
```
Comment (willesreis) [inline examples/mp-rest-jwt/README_pt.adoc:148]:
```suggestion
A classe `TokenUtils.java` é usada durante o teste para atuar como um servidor de Autorização que gera `Access Tokens` com base nos arquivos de configuração `privateKey.pem`, `publicKey.pem`, `Token1.json` e `Token2 .json`.
```

File: examples/mp-rest-jwt/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+
+`nimbus-jose-jwt` é a biblioteca usada para a geração JWT durante as provas.
```
Comment (willesreis) [inline examples/mp-rest-jwt/README_pt.adoc:150]:
```suggestion
`nimbus-jose-jwt` é a biblioteca usada para a geração do JWT durante os testes.
```

File: examples/mp-rest-jwt/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+  "auth_time": 1311280969,
+  "groups": [
+    "read-only"
+  ]
+}
+----
+
+== Cenários de teste
+
+`MovieTest.java` Contém 4 cenários OAuth2 para diferentes combinações de JWT.
```
Comment (willesreis) [inline examples/mp-rest-jwt/README_pt.adoc:197]:
```suggestion
`MovieTest.java` contém 4 cenários OAuth2 para diferentes combinações de JWT.
```

**Comment by Daniel-Dos** [comment]
@willesreis  and @cesarhernandezgt  for review .

File: examples/mp-rest-jwt/README_pt.adoc [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+    @Produces
+    JWTAuthContextInfo getContextInfo() throws InvalidKeySpecException, NoSuchAlgorithmException {
+        return getOptionalContextInfo().get();
+    }
+}
+----
+
+== Usando o MicroProfile JWT no TomEE
+
+O recurso JAX-RS `MoviesRest.java` contém vários contém vários endpoints que estão protegidos usando a anotação padrão `@ RolesAllowed`. O MicroProfile JWT é responsável por validar solicitações recebidas com o cabeçalho `Authorization' que fornece um token de acesso assinado.
```
Comment (willesreis) [inline examples/mp-rest-jwt/README_pt.adoc:72]: Here, stayed in double the words *contém vários*.
```suggestion
O recurso JAX-RS `MoviesRest.java` contém vários endpoints que estão protegidos usando a anotação padrão `@ RolesAllowed`. O MicroProfile JWT é responsável por validar solicitações recebidas com o cabeçalho `Authorization' que fornece um token de acesso assinado.
```

**Comment by Daniel-Dos** [comment]
thanks @willesreis for review.

**Comment by Daniel-Dos** [comment]
thanks

**Pull request merged**